cappy.py

```diff
@@ -123,11 +137,12 @@ def handle_inbound_request(self, message):
             request_id = message['id']
             args = message['args']
         except KeyError as e:
+            print(e)
```

Comment:
I was just trying to make it obvious that real error handling code should go there later, hence the TODO.